### PR TITLE
Fix some vulnerabilities and bump version

### PR DESCRIPTION
**Pull request by Jazper**
Before - 
`84 vulnerabilities (1 low, 51 moderate, 27 high, 5 critical)`

After - 
`15 vulnerabilities (8 moderate, 7 high)`

### CODE REVIEW

File: vue.config.js [hunk 1]
```diff
@@ -1,21 +1,12 @@
 module.exports = {
-  crossorigin: 'anonymous',
+  crossorigin: "anonymous",
```
Comment (th0r) [inline vue.config.js:2]: Why all single quotes are replaced with double?

**Review by th0r — CHANGES_REQUESTED** [review]

File: vue.config.js (hunk 1 above)
Comment (Jazper) [reply vue.config.js]: I'm so sorry. Prettier has double quotes by default. Changed it in the settings. I'll try to check the code better.

File: vue.config.js [hunk 2]
```diff
@@ -1,21 +1,12 @@
...
-    disableHostCheck: true
+    allowedHosts: 'all',
   },
   pluginOptions: {
     express: {
       shouldServeApp: true,
-      serverDir: './server'
-    }
+      serverDir: './server',
+    },
```
Comment (th0r) [inline vue.config.js:10]: Another unnecessary changes. Why do you process this project with prettier?

**Review by th0r — CHANGES_REQUESTED** [review]

File: vue.config.js (hunk 2 above)
Comment (Jazper) [reply vue.config.js]: Hm. This change I didn't missed. I thought these were improvements.
Removed.

**Review by th0r — APPROVED** [review]
